packages/world/remappings.txt

```diff
@@ -0,0 +1,3 @@
+ds-test/=../../node_modules/ds-test/src/
+forge-std/=../../node_modules/forge-std/src/
+store/=../../node_modules/@latticexyz/store/src/
```

Comment:
we're getting a lot of packages, some with really generic names
wdyt about a more verbose remapping, or even a full path?
(I kinda like how hardhat just doesn't have remappings, makes dependencies more verbose but much easier to understand)

Reply:
+1 to verbose/namespaced mapping, and would prefer to keep it tied to the npm namespace, i.e. `@latticexyz/store` for now and `@mud.dev/store` in the future